world/templates/migrations/0001_initial.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-10-31 01:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.db.models.manager
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('character', '0025_storyemit_orgs'),
+        ('objects', '0009_remove_objectdb_db_player'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Template',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('desc', models.TextField()),
+                ('access_level', models.CharField(choices=[(b'PR', b'PRIVATE'), (b'RS', b'RESTRICTED'), (b'OP', b'OPEN')], default=b'PR', max_length=2)),
+                ('attribution', models.CharField(max_length=60)),
+                ('apply_attribution', models.BooleanField(default=False)),
+                ('title', models.CharField(max_length=255)),
+                ('applied_to', models.ManyToManyField(blank=True, to='objects.ObjectDB')),
+            ],
+            options={
+                'abstract': False,
+            },
+            managers=[
+                ('accessible', django.db.models.manager.Manager()),
```

Comment:
Changing 'accessible' to 'objects' would change the migration here, I think. I can't recall ever seeing the managers kwarg in CreateModel before, so not entirely sure if it'd just be 'objects', or removed as a default, or what.

Reply:
Resolved with new migration (from push).